### DISCUSSION TITLE
공식 샘플3번에서 상자를 열때와 같은 구조의 ui가 모바일기기에서 안뜹니다.

### DISCUSSION THREAD

**Discussion by HanjuKim786** [naverz/zepeto-studio-kor#881] [5. Q&A]
유니티상에서는 정상적으로 뜨는데 실제 모바일 환경에서는 뜨지 않습니다.

캐릭터가 가까이 가면 SetActive(true) 로 보이고 안보이고를 조절하고 있습니다.
실제 구현은 의자를 두고 의자와 상호작용하는 것을 목적으로 해당 UI를 제작했습니다.
모바일에서만 안보이는데에 특별한 이유가 있을까요?

예제3번은 실제로 퍼블리시하고 해당 맵을 모바일기기에서 테스트한 코드인가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 이슈가 발생한 월드 버전 및 제페토 앱 버전을 알려주시고
현재 known 이슈인 getComponent를 사용한 코드는 없는지도 올려주시면 파악에 도움이 될 것 같습니다.
https://github.com/naverz/zepeto-studio-kor/discussions/528

**Reply by HanjuKim786** [reply]
안녕하세요
월드 버전 : 1.3.0
제페토 앱 버전 : 3.13.5

GetComponent를 사용하고 있습니다.

**Reply by Zepeto-TechEvangelist** [reply]
네, 모바일에서만 동작하지 않는다면 GetComponent에 따른 이슈로 보여집니다.
다음 월드 업데이트때 해당 이슈가 수정될 예정입니다.
